Dockerfile

```diff
@@ -18,5 +18,8 @@ WORKDIR /stripped_repo
 # Create gradle cache
 # `rw` option doesn't allow to make any changes on original dir but rather create something like overlayfs
 # We need this to allow `./gradlew` to write in `.gradle` directory
+# script for enforcing compatifility is runned for download IntelliJ dependencies
```

Comment:
```suggestion
# Script for enforcing compatibility is being run to download IntelliJ dependencies
```
Also, this can be moved to right before line 25